packages/ts-codegen/src/__tests__/index.test.ts

```diff
@@ -489,15 +489,15 @@ describe(generateTS, () => {
             export type id = (model: UserModel, args: {}, context: unknown, info: ResolveInfo) => PromiseOrValue<string>;
         }
         export type PresenceInput = {
-            type: string;
+            readonly type: string;
```

Comment:
These types are only used in resolvers and thus only ever read, right?

Reply:
Correct.